examples/kitchen-sink/packages/ui/tsup.config.ts

```diff
@@ -0,0 +1,13 @@
+import { defineConfig } from "tsup";
+
+export default defineConfig({
+  entry: {
+    ".": "src/index.tsx",
+  },
+  banner: {
+    js: "'use client'",
+  },
```

Comment:
This works, but it's a little confusing as it's now dictating the type of components we can put in this package. And it makes the decision between bundling / using an internal package more nuanced. Not a change request, just a comment!

Reply:
It's definitely weird but I poked around the ecosystem and it doesn't look like there is an idiomatic way that people handle this yet. It's also notable that it's generally an okay approach as we're all learning how to migrate together. 😬